source/_components/light.limitlessled.markdown

```diff
@@ -74,6 +77,9 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
   - When using a legacy WiFi bridge (before v6), you can observe on the MiLight mobile application, you can not select a specific brightness or temperature - you can only step each property up or down. There is no indication of which step you are on. This restriction, combined with the unreliable nature of LimitlessLED transmissions, means that setting white bulb properties is done on a best-effort basis. The only very reliable settings are the minimum and maximum of each property.
   - *Temperature*: Wifi bridge v6 supports 101 temperature steps; older versions only 10.
   - *Brightness*: Wifi bridge v6 supports 101 brightness steps; older versions only 10.
+- **Dimmer**
+    - This type is for a single color LED dimmer like the 1CH MiLight dimmer module or similar. Version 6 Wifi bridges are supported and not sure if legacy Wifi bridges(before v6) are working with this type.
```

Comment:
Please remove " and not sure if legacy Wifi bridges(before v6) are working with this type" since it won't work in HA at least (no legacy command set was added).

Reply:
So only version 6 of the bridge is supported, and support for the older versions is dropped? Because the rest of the documentation is talking about the legacy wifi bridges... I only have an V6 bridge with one LED controller.

Reply:
The old bridges only support lights that existed at that time, AFAIK.

This is similar to RGBWW which has a note "(Only supported on v6 bridges)" so actually you could remove the entire last sentence and add that note in a similar way, for consistency.

Reply:
Broadly speaking, `python-limitlessled` and therefore HASS support legacy (v5 and earlier) and current (v6) bridges. But support for the 1CH dimmer is only available for v6 bridges (since your PR added a V6 command set, not a Legacy command set). v5 doesn't have dimmers anyway, I don't think.

Reply:
Okay, thanks for the fast explanation. I will edit this tomorrow and update the PR.